Cognite.Jetfire.Cli/JetfireRootCommand.cs

```diff
@@ -2,33 +2,31 @@
 // SPDX-License-Identifier: Apache-2.0
 
 using System;
+using System.Collections.Generic;
 using System.CommandLine;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Cognite.Jetfire.Cli
 {
     public class JetfireRootCommand
     {
-        public JetfireRootCommand(
-            Deploy.DeployCommand deployCommand,
-            Query.QueryCommand queryCommand,
-            Run.RunCommand runCommand,
-            ListTransforms.ListCommand listCommand
-        )
+        public JetfireRootCommand(IList<IJetfireSubCommand> subCommands)
```

Comment:
I would consider taking an IEnumerable here, but that's not important

Reply:
Ah, nice one. I'll do that in the next PR!